ocs_ci/ocs/machine.py

```diff
@@ -262,6 +262,24 @@ def add_node(machine_set, count):
     return True
 
 
+def delete_node(machine_set, count):
+    """
+    Delete nodes from the cluster by decreasing the number of replicas in the
+    machine set.
```

Comment:
please add an empty line between the docstring's sections

Reply:
done.